### PR TITLE
Added myclirc option to prevent warnings from occurring.

### LINKED ISSUE #110: Alter 'Destructive Query' Warning to be optional
As analysts, we need a way to turn that off. We do a lot of them all day.  This is in progress by me.

### PR DESCRIPTION
**Pull request by danieljwest**
Resolves #110 

Just added a myclirc section for destructive warning with the default being True. 
Let me know if there's anything else you would like added or if I made any glaring mistakes.

### CODE REVIEW

File: mycli/myclirc [hunk 1]
```diff
@@ -11,6 +11,10 @@ smart_completion = True
 # lines. End of line (return) is considered as the end of the statement.
 multi_line = False
 
+# Destructive warning mode will alert you before executing a sql statement
+# that may cause harm to the database.
```
Comment (amjith) [inline mycli/myclirc]: Let's add examples for destructive commands. Like `drop database` or `shutdown`.

File: mycli/main.py [hunk 2]
```diff
@@ -346,15 +347,15 @@ def prompt_tokens(cli):
...
-                    continue
+                if self.destructive_warning:
+                    destroy = confirm_destructive_query(document.text)
+                    if destroy is None:
+                        pass  # Query was not destructive. Nothing to do here.
+                    elif destroy is True:
+                        self.output('Your call!')
+                    else:
+                        self.output('Wise choice!')
+                        continue
```
Comment (amjith) [inline mycli/main.py]: Mixed spaces and tabs is a bad idea in a language that is whitespace sensitive. 

Most python projects use 4 spaces for indentation.

Comment (amjith) [inline mycli/main.py]: Sorry I misunderstood those highlighted diffs as tabs. My bad.

**Comment by amjith** [comment]
The change looks good. Let's add some examples in the config file and we're good to merge.

File: mycli/main.py (hunk 2 above)
Comment (danieljwest) [inline mycli/main.py]: No worries. I was pretty confused! I had set my IDE to not use idents and was trying to figure out where and how I inserted tabs.

**Comment by danieljwest** [comment]
Examples updated!

**Comment by amjith** [comment]
:+1: 

Thanks for the PR. 

:mushroom:

**Pull request merged**